owl_codegen/lib/json_generator.dart

```diff
@@ -124,7 +124,14 @@ class _Field {
   _Field.fromElement(FieldElement elem) {
     fieldName = elem.name;
     keyName = elem.name;
-    final jsonField = getAnnotation(elem, JsonField);
+    bool isGetterSetter = elem.getter != null && elem.setter != null;
+    final jsonField = isGetterSetter
```

Comment:
This is a bit hard to read, could you please unpack it into multiple statements?